docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,67 @@
-[//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
-
 # Connecting to a local deployment
```

Comment:
```suggestion
# Connect to a local deployment
```

We decided to lose the gerunds (the "ing") for the titles in GDK docs. Just noticed this!